### ISSUE TITLE
flake8 not respecting excluded

### ISSUE THREAD

**Issue by krzysztof-grobelny** [kevinglasson/vscode-cornflakes-linter#2]
I've got cornflakes installed as a VS Code plugin on MacOS.
Config (.flake8 file in the project dir) is as follows:
```
[flake8]
ignore = E722,E501
max-line-length = 120
exclude = .vscode,.pyenv,builtins.pyi,case.pyi,mock.pyi
```
All excludes here are results of the struggle with the following behavior (screenshot):
https://ibb.co/0XqxFDj

Basically flake8 is analyzing all imports, no matter how odd path to them is.
I'm trying to filter out path to that type of imports (.vscode,.pyenv) or the actual imported file by its name (builtins.pyi, case.pyi, mock.pyi). Without any luck.

The result is that I've got 900 not 30 issues reported by flake8.

**Comment by kevinglasson** [comment]
I'm sorry I didn't get back to you much sooner! You've probably moved on to another solution - or are just ignoring it!

I don't seem to be able to replicate this problem, I'm also on MacOS. I tried introducing a lint error (which I could see) and then added it to the exclude field in the project roots .flake8 and saved the file and the 'problem' disappeared. Note: I did need to save the file again to get the problems tab to update.
